tool/gravity/cli/operation.go

```diff
@@ -99,7 +99,10 @@ func completeOperationPlan(localEnv, updateEnv, joinEnv *localenv.LocalEnvironme
 		// There's only one install operation
 		return completeInstallPlan(localEnv, op)
 	case ops.OperationExpand:
-		return completeJoinPlan(localEnv, joinEnv, op)
+		if op == nil {
+			return trace.BadParameter("operation ID must be specified")
```

Comment:
Nit: Maybe require this only if multiple operations are in progress? I.e. above find all active operations, and if there're more than 1 and operationID is empty then bounce, otherwise proceed as before.

Reply:
That code wrong anyways - I've restructured operation handling so it's uniform for all operation types.